packages/foreman/nodejs-datatables.net-bs/nodejs-datatables.net-bs.spec

```diff
@@ -27,7 +25,9 @@ Requires: npm(jquery) >= 1.7
 
 %install
 mkdir -p %{buildroot}%{nodejs_sitelib}/%{npm_name}
-cp -pfr Readme.md css js package.json %{buildroot}%{nodejs_sitelib}/%{npm_name}
```

Comment:
Just checking, there is no need for the Readme.md anymore?

Reply:
I suppose not. This was auto-generated.